Clients/GUI/Pages/SignUpWizzard.razor

```diff
@@ -0,0 +1,170 @@
+            </div>
+            <div class="form-group">
+                <label for="Surname">Surname:</label>
+                <InputText class="form-control col-3" @bind-Value="userInput.LastName" placeholder="Your surname" />
+            </div>
+            <div class="form-group">
+                <label for="Birthday">Birthday:</label>
+                <InputDate class="form-control col-3" @bind-Value="userInput.Birthday" />
+            </div>
+            <div class="form-group">
+                <label for="Email">Email</label>
+                <InputText class="form-control col-3" @bind-Value="userInput.Email" placeholder="example@gmail.com" />
+            </div>
+            <div class="form-group">
+                <label for="Password">Password</label>
+                <InputText class="form-control col-3" type="password" @bind-Value="userInput.Password" placeholder="*****" />
+            </div>
+
+            <div class="form-group">
+                <label for="ConfirmPassword">Confirm Password</label>
+                <InputText class="form-control col-3" type="password" @bind-Value="userInput.ConfirmPassword" placeholder="*****" />
+            </div>
+            <br />
+
+            <ValidationSummary />
+            @if (isErrorDisplayed)
+            {
+                <label style=" color: red;"> @serverErrorMessages </label>
+                <br />
+            }
+
+        </WizardStep>
+        <WizardStep Name="Second Step">
+            <div class="row">
+                <div class="container m-5">
+                    <label>Send your avatar</label>
+                    <br />
+                    <label for="files" class="btn btn-primary">Select Image</label>
+                    <InputFile style="display: none" id="files" accept=".jpg, .jpeg, .png" OnChange="HandleSelection" />
+                    <br />
+                    <label>@fileName</label>
+                    <br />
+                    @if (errorMessageFile.Length != 0)
+                    {
+                        <label style="color: red;"> @errorMessageFile </label>
+                        <br />
+                    }
+                    <button class="btn btn-secondary btn-sm" type="button" @onclick="ClearFile">clear</button>
+                    <br />
+                    @if (isErrorDisplayed)
+                    {
+                        <label style="color: red;"> @serverErrorMessages </label>
+                        <br />
+                    }
+                </div>
+            </div>
+            @if (showLoading)
+            {
+                <div class="spinner-border" role="status">
+                    <span class="sr-only">Loading...</span>
+                </div>
+            }
+        </WizardStep>
+    </Wizard>
+</EditForm>
+
+@code {
+    bool isErrorDisplayed = false;
+    string serverErrorMessages = "";
```

Comment:
May be name _message_ would be better, because it stores only one message at a time as I understood

Reply:
fixed in future commit